htdocs/GetCSS.php

```diff
@@ -93,6 +95,20 @@
     $FullPath = "$ModuleDir/css/$File";
 }
 
+$public = false;
```

Comment:
$public = true.

Reply:
the default should be false for backwards compatibility since it's what the behaviour was before public modules was implemented (and it's a more secure default)